crates/transaction-pool/src/test_utils/mock.rs

```diff
@@ -516,8 +527,8 @@ impl TransactionOrdering for MockOrdering {
     type Priority = U256;
     type Transaction = MockTransaction;
 
-    fn priority(&self, transaction: &Self::Transaction) -> Self::Priority {
-        transaction.gas_cost()
+    fn priority(&self, transaction: &Self::Transaction, base_fee: u64) -> Self::Priority {
+        U256::from(transaction.effective_tip_per_gas(base_fee).expect("tx has been validated"))
```

Comment:
this is now dynamic

I don't think we update this when the base fee changes?
this would require to reorder the all txs in the pool, we don't do this atm?